packages/blockchain-wallet-v4-frontend/src/modals/Withdraw/EnterAmount/template.success.tsx

```diff
@@ -103,6 +104,14 @@ const Success: React.FC<InjectedFormProps<
 
   const showPendingTransactions = !isEmpty(props.locks)
 
+  const showInfoTooltip =
+    Number(props.withdrawableBalance) < Number(props.availableBalance)
+
+  // eslint-disable-next-line
+  console.log('propssss', props)
+  // eslint-disable-next-line
+  console.log('showInfoTooltip', showInfoTooltip)
```

Comment:
remove these logging lines

Reply:
@milan-bc you shouldn't need //eslint-disable-next-line for running locally, so you can still console.log w/o adding eslint-disable.

When you try to commit however, commitlint should warn you about the logs, so you can remove them before comitting and pushing.

Reply:
@plondon cool, will check 👍